test/playback.test.js

```diff
@@ -59,6 +59,27 @@ QUnit.test('Advanced Bip Bop', function(assert) {
   });
 });
 
+QUnit.skip('Fmp4', function(assert) {
```

Comment:
I think it would be worth specifying that this is a playlist with both fmp4 and ts segments, as we do support fmp4 alone right now.